.github/workflows/ci.yaml

```
@@ -191,6 +191,12 @@ jobs:
         run: |
           make --output-sync=line -j lint
 
+      - name: Check workflow files
+        run: |
+          bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/7fdc9630cc360ea1a469eed64ac6d78caeda1234/scripts/download-actionlint.bash)
```

```suggestion
          bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash) 1.7.1
```
This will make it easier to bump versions.
ref: https://github.com/rhysd/actionlint/blob/main/docs/install.md#download-script

---

Good idea 👍 I've pinned to `1.6.22` which is the version to which the current commit is pointing.

---

I suggest we pin to latest availbale just to be sure that linter supports any newly added syntax.

---

If we do so, we should update code-server too. See https://github.com/coder/code-server/blob/main/.github/workflows/build.yaml#L142-L152.

@kylecarbs any objections?